docs/design/services/dynamodb/high-level-library/prototype/option-1/sync/Prototype.java

```diff
@@ -0,0 +1,1025 @@
+/**
+ * The entry-point for all DynamoDB client creation. All Java Dynamo features will be accessible through this single class. This
+ * enables easy access to the different abstractions that the AWS SDK for Java provides (in exchange for a bigger JAR size).
+ *
+ * <p>
+ *     <b>Maven Module Location</b>
+ *     This would be in a separate maven module (software.amazon.awssdk:dynamodb-all) that depends on all other DynamoDB modules
+ *     (software.amazon.awssdk:dynamodb, software.amazon.awssdk:dynamodb-document). Customers that only want one specific client
+ *     could instead depend directly on the module that contains it.
+ * </p>
+ */
+@ThreadSafe
+public interface DynamoDb {
```

Comment:
Are we going to have an `AsyncDynamoDb` which contains async clients or the low level async clients will be in this interface?

Reply:
I was thinking this would also contain the async clients, so that there's only one specific entry point.